pkg/remote/aws/api_gateway_account_enumerator.go

```diff
@@ -0,0 +1,51 @@
+package aws
+
+import (
+	"github.com/cloudskiff/driftctl/pkg/remote/aws/repository"
+	remoteerror "github.com/cloudskiff/driftctl/pkg/remote/error"
+	"github.com/cloudskiff/driftctl/pkg/resource"
+	"github.com/cloudskiff/driftctl/pkg/resource/aws"
+)
+
+type ApiGatewayAccountEnumerator struct {
+	repository repository.ApiGatewayRepository
+	factory    resource.ResourceFactory
+}
+
+func NewApiGatewayAccountEnumerator(repo repository.ApiGatewayRepository, factory resource.ResourceFactory) *ApiGatewayAccountEnumerator {
+	return &ApiGatewayAccountEnumerator{
+		repository: repo,
+		factory:    factory,
+	}
+}
+
+func (e *ApiGatewayAccountEnumerator) SupportedType() resource.ResourceType {
+	return aws.AwsApiGatewayAccountResourceType
+}
+
+func (e *ApiGatewayAccountEnumerator) Enumerate() ([]*resource.Resource, error) {
+	account, err := e.repository.GetAccount()
+	if err != nil {
+		return nil, remoteerror.NewResourceListingError(err, string(e.SupportedType()))
+	}
+
+	results := make([]*resource.Resource, 1)
+
+	if account != nil {
+		attrs := map[string]interface{}{}
+		if account.CloudwatchRoleArn != nil && *account.CloudwatchRoleArn != "" {
+			attrs["cloudwatch_role_arn"] = *account.CloudwatchRoleArn
```

Comment:
Why do we need this attribute ?

Reply:
We could totally ditch it, it was more for preparing an eventual "deep-mode" compatibility since there is only one attribute in this resource that is needed by the provider.

![image](https://user-images.githubusercontent.com/8110579/134943717-76a01896-f078-4d87-b21d-b3cfc459401f.png)

Reply:
Yeah I think we should not add useless code for now since it is only a deep mode required field.